### ISSUE TITLE
$ edge case

### ISSUE THREAD

**Issue by GiselleSerate** [GiselleSerate/myaliases#54]
What if you want to use a variable in your nickname but you want it evaluated on src instead of on calling altalias? Use single quotes: `'$var1'="commands"`

Use cases: Uh . . . call with username?

Consider whether you even want this behavior or if someone would find this useful. It's undefined by alias.

**Comment by aryam7** [comment]
Hey, also I'm still kinda confused about something: what was the filter originally for?
Because `alias $var1='cds'` and `alias $(echo 'cd')='cds'` should work without filtering, right? I didn't have to put double quotes around those for them to work. As long as there are double quotes around `$1` it should be fine, right?
Doesn't bash evaluate strings with a $ sign before they're handed over to the alias command?

**Comment by GiselleSerate** [comment]
Yeah, I guess you're right . . . the filter currently only removes the `=` because I left it in my regex. So I guess as long as the alias name expression doesn't involve assignment, the filter should work fine.

**Comment by aryam7** [comment]
Yeah, and `=` is not a valid character in alias names anyway.

**Comment by GiselleSerate** [comment]
Well, I used regex to isolate the alias nickname bit so I could call alias again to figure out the expression it stored. But I didn't do regex to change what the person used as their nickname.

**Comment by aryam7** [comment]
ah ok makes sense
and once you have that alias nickname, you use it to delete a line in .usr_aliases that already declares that alias, right?
